app/javascript/dashboard/composables/store.js

```diff
@@ -17,7 +17,7 @@ export const useStoreGetters = () => {
   );
 };
 
-export const mapGetter = key => {
+export const useMapGetter = key => {
```

Comment:
@pranavrajs I've checked, this is not used elsewhere, renaming it is fine. It's a good shorthand